package.js

```diff
@@ -11,7 +11,6 @@ Package.onUse(function(api) {
 
   api.use([
     'underscore',
-    'templating',
```

Comment:
This change is important since it causes: 
`error: conflict: two packages included in the app (templating and peerlibrary:blaze-components) are both trying to handle *.html`